jerry-core/parser/js/byte-code.c

```diff
@@ -21,6 +21,15 @@ JERRY_STATIC_ASSERT ((sizeof (cbc_uint8_arguments_t) % sizeof (jmem_cpointer_t))
 JERRY_STATIC_ASSERT ((sizeof (cbc_uint16_arguments_t) % sizeof (jmem_cpointer_t)) == 0,
                      sizeof_cbc_uint16_arguments_t_must_be_divisible_by_sizeof_jmem_cpointer_t);
 
+/**
+ * The reason of these two static asserts to notify the developer to increase the JERRY_SNAPSHOT_VERSION
+ * whenever new bytecodes are introduced or existing ones have been deleted.
+ */
+JERRY_STATIC_ASSERT (CBC_END == 238,
+                     number_of_cbc_opcodes_must_be_238);
+JERRY_STATIC_ASSERT (CBC_EXT_END == 115,
+                     number_of_cbc_opcodes_must_be_115);
```

Comment:
I would rename this to `number_of_cbc_opcodes_changed` and `number_of_cbc_ext_opcodes_changed` so less number of lines needs to be updated. (Btw why the second is `number_of_cbc_opcodes_must_be_115` and not `number_of_cbc_ext_opcodes_must_be_115`)

Reply:
It was a typo, fine!